docs/coordinates/spectralcoord.rst

```diff
@@ -0,0 +1,530 @@
+
+Note that these resulting velocities are different from the ``radial_velocity``
+property (which is still zero here) - the latter is the difference in velocity
+between observer and target, while the former are how much the spectral values
+are Doppler shifted by relative to the rest frequency or wavelength.
+
+So if the features are indeed from 12CO, they have velocities of approximately -11.9, -27.5 and
+-30.1 km/s relative to the T tau rest frame.
+
+.. _spectralcoord-common-frames:
+
+Common velocity frames
+======================
+
+Any valid astropy coordinate frame can be passed to the
+:meth:`~astropy.coordinates.SpectralCoord.with_observer_velocity`
+method, including string aliases such as ``icrs``. Below we list some of the
+frames commonly used to define spectral coordinates in:
+
+The velocity frames available as constants on the |SpectralCoord| class are:
+
+========================== =================================================
+Frame name                 Description
+========================== =================================================
+``'gcrs'``                 Geocentric frame (defined as stationary relative to the GCRS origin)
+``'icrs'``                 Barycentric frame (defined as stationary relative to the ICRS origin)
+``'hcrs'``                 Heliocentric frame (defined as stationary relative to the HCRS origin)
+``'lsrk``                  Kinematic Local Standard of Rest (LSRK),
+                           defined as having a velocity of 20 km/s towards
+                           18h +30d (B1900) relative to the Solar System
+                           Barycenter [1]_.
+``'lsrd'``                 Dynamical Local Standard of Rest (LSRD),
+                           defined as having a velocity of U=9 km/s,
+                           V=12 km/s, and W=7 km/s in Galactic coordinates
+                           (equivalent to 16.552945 km/s towards l=53.13
+                           and b=25.02) [2]_.
+``'lsr'``                  A more recent definition of the Local Standard
+                           of rest, with U=11.1 km/s,
+                           V=12.24 km/s, and W=7.25 km/s in Galactic coordinates [3]_.
+========================== =================================================
+
+References
+==========
+
+.. [1] Meeks, M. L. 1976, *Methods of experimental physics. Vol._12.
+       Astrophysics. Part C: Radio observations*, Section 6.1 by Gordon, M. A.
+       `[ADS] <https://ui.adsabs.harvard.edu/abs/1976mep..book.....M>`__.
+.. [2] Delhaye, J. 1965, *Galactic Structure*. Edited by Adriaan Blaauw and
+       Maarten Schmidt. Published by the University of Chicago Press, p61
+       `[ADS] <https://ui.adsabs.harvard.edu/abs/1965gast.book...61D>`__.
+.. [3] Schönrich, R., Binney, J., & Dehnen, W. 2010, MNRAS, 403, 1829
+       `[ADS] <https://ui.adsabs.harvard.edu/abs/2010MNRAS.403.1829S>`__.
+
+.. The following frames are defined in FITS WCS and may be added here in future:
```

Comment:
How about uncommenting this and giving an example of how one would transform to one of these oneself?

Reply:
Done - I added an example of the local group velocity frame